cli/cmd/brokers.go

```diff
@@ -1,16 +1,38 @@
 package cmd
 
 import (
+	"bytes"
+	"encoding/json"
 	"fmt"
+	"io/ioutil"
+
+	"github.com/cncf/servicebroker/k8s/service_controller/model"
 	"github.com/spf13/cobra"
 )
 
+const (
+	BROKERS_URL     = "/v2/service_brokers"
+	BROKERS_FMT_URL = "/v2/service_brokers/%s"
+)
+
+var (
+	brokerURL string
+	user      string
+	password  string
+)
+
 func init() {
 	RootCmd.AddCommand(brokersCmd)
-	brokersCmd.AddCommand(createCmd)
-	brokersCmd.AddCommand(describeCmd)
-	brokersCmd.AddCommand(listCmd)
-	brokersCmd.AddCommand(deleteCmd)
+	brokersCmd.AddCommand(createBrokersCmd)
+	brokersCmd.AddCommand(describeBrokersCmd)
+	brokersCmd.AddCommand(listBrokersCmd)
+	brokersCmd.AddCommand(deleteBrokersCmd)
+	createBrokersCmd.Flags().StringVarP(&name, "name", "n", "", "Name to give to Service Broker")
+	createBrokersCmd.Flags().StringVarP(&brokerURL, "brokerURL", "b", "", "URL of the broker")
+	createBrokersCmd.Flags().StringVarP(&user, "user", "u", "", "user name to use for broker auth")
+	createBrokersCmd.Flags().StringVarP(&password, "password", "p", "", "password to use for broker auth")
+	createBrokersCmd.Flags().StringVarP(&spaceGUID, "spaceGUID", "s", "", "Space GUID to use for broker")
```

Comment:
Is it worth removing all space references? It seems like they're unused and unnecessary in the generic model.